modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,10 +1,306 @@
+
+| https://issues.couchbase.com/browse/MB-48146[MB-48146^]
+| *Summary*: [IPV6] no index available when switching to ipv6 or ipv6only IP family
+
+| https://issues.couchbase.com/browse/MB-48336[MB-48336^]
+| *Summary*: Persist rollbacks stats in case of indexer crash - needed for verification
+
+| https://issues.couchbase.com/browse/MB-48366[MB-48366^]
+| *Summary*: [Enforce-TLS] 'Rebalance exited with reason {service_rebalance_failed,index}..'
+
+|===
+
+==== Search Service
+
+[#table_fixedissues_v702-search,cols="25,66"]
+|===
+|Issue | Description
+
+
+| https://issues.couchbase.com/browse/MB-46260[MB-46260^]
+| *Summary*: Apply RBAC only for target collections in a multi-collection index
+
+|===
+
+==== Eventing Service
+
+[#table_fixedissues_v702-eventing,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-46351[MB-46351^]
+| *Summary*: dcp_stream_boundary remains as "from_prior" after upgrade
+
+| https://issues.couchbase.com/browse/MB-46647[MB-46647^]
+| *Summary*: default value for language_compatibility should be 6.6.2 instead 6.5.0
+
+| https://issues.couchbase.com/browse/MB-47867[MB-47867^]
+| *Summary*: Always emit the first exception to the application log
+then summarize
+
+| https://issues.couchbase.com/browse/MB-48103[MB-48103^]
+| *Summary*: handler stuck in deploying state
+
+| https://issues.couchbase.com/browse/MB-48104[MB-48104^]
+| *Summary*: Connection::isPacketAvailable(): Invalid packet header detected. Cookies: []
+
+| https://issues.couchbase.com/browse/MB-48118[MB-48118^]
+| *Summary*: Performance: crash dump generated
+
+| https://issues.couchbase.com/browse/MB-48195[MB-48195^]
+| *Summary*: REST calls fail after changing encryption level to "all"
+
+| https://issues.couchbase.com/browse/MB-48337[MB-48337^]
+| *Summary*: Mix Mode: No error when we add new function
+
+| https://issues.couchbase.com/browse/MB-48487[MB-48487^]
+| *Summary*: panic observed in debugger tests
+
+| https://issues.couchbase.com/browse/MB-48488[MB-48488^]
+| *Summary*: Resume time increased by 34%
+
+| https://issues.couchbase.com/browse/MB-48572[MB-48572^]
+| *Summary*: Service 'eventing' exited with status 2.
+
+|===
+
+==== Cross Data Center Replication (XDCR)
+
+[#table_fixedissues_v702-xdcr,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-47157[MB-47157^]
+| *Summary*: XDCR - make health_check_interval configurable
+
+| https://issues.couchbase.com/browse/MB-47246[MB-47246^]
+| *Summary*: Switch to new javascript evaluator
+
+| https://issues.couchbase.com/browse/MB-47521[MB-47521^]
+| *Summary*: XDCR - collect remote clusters and replication info as part of cbcollect
+
+| https://issues.couchbase.com/browse/MB-47546[MB-47546^]
+| *Summary*: Bind only to IPv4 addresses when invoked with IPv4-Only cluster-wide setting - xdcr
+
+| https://issues.couchbase.com/browse/MB-47723[MB-47723^]
+| *Summary*: Enforce TLS - XDCR
```

Comment:
Remove this one, it's a new feature, not an issue

Reply:
Removed.